docs/advanced.md

```diff
@@ -69,6 +69,12 @@ each request. Each of the keys in the context added by this integration will be
 
 To enable the FastAPI integration, you have to modify the `FastAPI` app instance.
 
+There are two ways to do this. The first, chipperize() is the recommended approach to get accurate timing data
+in most cases-- it ensures the Woodchipper middleware is the first in the middleware stack.
+However, if you also have Datadog's DDTrace middleware installed on the same application, chipperize may cause
+tracing to break. In those cases, manual installation (see second example) is preferred.
+
+Chipperize:
 {{< highlight python "lineNos=true,anchorLineNos=true,lineAnchors=flask" >}}
```

Comment:
I don't think the highlight markdown syntax is working right now-- the preview is rendering it as plaintext. Would it be ok to switch to a standard code block with syntax highlighting? Or does anyone else know how to get it to work?